### PR TITLE
Adding localStorageDB 2.3.1

### PR DESCRIPTION
**Pull request by a904guy**
http://nadh.in/code/localstoragedb/
https://github.com/knadh/localStorageDB

### CODE REVIEW

**Comment by a904guy** [comment]
Shit, was updating my fork...

**Comment by vvakame** [comment]
*localstoragedb/localstoragedb.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/localstoragedb
  * http://bower.io/search/?q=localstoragedb
  * others?
* [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (localstoragedb/localstoragedb-tests.ts or others)
* [ ] pass the Travic-CI test?

File: localstoragedb/localstoragedb.d.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	 Returns the numerical ID if a new row was inserted, or an array of IDs if rows were updated
+	 */
+	deleteRows(table:string,query:localStorageDB_dynamicFields|localStorageDB_callbackFilter): number;
+	/*
+	 Deletes rows from a table matching query, and returns the number of rows deleted
+	 - query is either an object literal or a function. If query is not supplied, all rows are deleted
+	 */
+}
+
+interface localStorageDB_fields<T> {
```
Comment (vvakame) [inline localstoragedb/localstoragedb.d.ts]: you should use `namespace`.

**Comment by a904guy** [comment]
Alright, I'll setup a test file, and adjust the file name. Will be a bit later in the day.

File: localstoragedb/localstoragedb.d.ts (hunk 1 above)
Comment (a904guy) [inline localstoragedb/localstoragedb.d.ts]: Ok, I'll make the adjustment. Thank you.

**Pull request closed without merging**